lib/statement.js

```diff
@@ -64,9 +65,31 @@ class Statement {
    * @memberof Statement
    */
   async bind(params) {
+    deprecate('As of 1.1.0, bind() is deprecated use bindParameters() instead.');
```

Comment:
FYI since bindParam also throws a deprecation warning, this would result in double warnings. Not sure if good or bad. Would have to duplicate bindParam here or separate the body of bindParam in to an internal function and make both wrappers on top of the new function.

Reply:
Yes I will create a `bindParamWrapper()` that both `bindParam()` and `bind()` will call.